lib/vtls/openssl.c

```diff
@@ -3774,7 +3774,12 @@ static size_t Curl_ossl_version(char *buffer, size_t size)
 {
 #ifdef OPENSSL_IS_BORINGSSL
   return msnprintf(buffer, size, OSSL_PACKAGE);
-#else /* OPENSSL_IS_BORINGSSL */
+#elif defined(HAVE_OPENSSL_VERSION) && defined(OPENSSL_VERSION_STRING)
+  return msnprintf(buffer, size, "%s/%s",
+                   OSSL_PACKAGE, OpenSSL_version(OPENSSL_VERSION_STRING));
+#else
+  /* OPENSSL_IS_BORINGSSL */
```

Comment:
This new placement of the comment seems odd, it makes it seem like we are in the `OPENSSL_IS_BORINGSSL` block, when it's really the opposite?

Reply:
ack, I'll fix before I merge!